src/platform/windows/display_base.cpp

```diff
@@ -156,6 +174,14 @@ namespace platf::dxgi {
           BOOST_LOG(error) << "Unrecognized capture status ["sv << (int) status << ']';
           return status;
       }
+
+      if (config::video.unpaced) {
+        dup.release_frame();
```

Comment:
Return value is being discarded here. We should return here if `!platf::capture_e::ok && !platf::capture_e::timeout`?

Reply:
Done https://github.com/LizardByte/Sunshine/pull/1203/commits/677282d191d6bbf185404d6f2824bb500c40fede